test/scripts/box/box.js

```diff
@@ -251,7 +236,7 @@ describe('Box', () => {
     ]);
     await box.process();
 
-    processor.calledOnce.should.eql(true);
+    processor.calledOnce.should.to.be.true;
```

Comment:
```suggestion
    processor.calledOnce.should.be.true;
```

Since we are using `chai` to make our unit test into human language, we should consider about grammar things. There is no `to` after `should` in English, and chai could still work without `.to`.

Reply:
My personal preference is to reduce unnecessary chains, so I would like to agree with your review, but if you use the should interface, you should follow the grammar as much as possible.

Reply:
@segayuu 

From chai's documents, I can not see any example of `should.to.be`. It seems that `to.be` should only be used in `expect()` syntax (`expect().to.be`). And with `should` syntax, `should.be` will be enough.

Reply:
Certainly, the scene used in the should interface on the official website could not be confirmed. Although it is incomplete in English, the difference will be changed along the official website.